content/docs/get-started/index.md

```diff
@@ -216,84 +194,74 @@ y_pred = model.predict_proba(x)
 print(y_pred)
 ```
 
-Now, let's run the script:
+You can run it to print the prediction probabilities:
 
 ```cli
 $ python predict.py
 [[0.47 0.24 0.29]]
 ```
 
-We see that the prediction probabilities were successfully printed to stdout.
-
 </tab>
 
-<tab title="Command Line">
+<tab title="Command line">
 
 ### Batch scoring
 
-The MLEM CLI allows us to natively use any saved model directly for prediction
-or batch scoring with any local dataset. This is very handy if we want to get
-some quick feedback about a model we just created.
-
-First, create an example dataset file to apply, we'll go with a `csv` format:
+`mlem` lets you use any saved model natively for prediction or batch scoring
+with any dataset. This is very handy if you want to get quick feedback about a
+model, for example. Let's create a sample dataset and apply it to the `rf` model
+we saved earlier:
 
-```cli
-$ echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
-0,1,2,3" > new_data.csv
+```csv
+sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3
```

Comment:
The fact it's csv is not reflected and it took me some time to understand what is this. I'd suggest (part 1/2)

```suggestion
```csv
# new_data.csv
sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
0,1,2,3
```

Reply:
Agree. I'll move the file name to the previous paragraph because `csv` doesn't have comments like that (looks odd)...